tests/test_tzlocal.py

```diff
@@ -0,0 +1,171 @@
+import os
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+from unittest.mock import MagicMock, Mock
+
+import pytest
+
+import tzlocal.unix
+import tzlocal.utils
+
+if sys.version_info >= (3, 9):
+    from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+else:
+    from backports.zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+
+
+@pytest.fixture(scope='session', autouse=True)
+def clear_tz_env_variable():
+    os.environ.pop('TZ', None)
+
+
+@pytest.fixture
+def tz_path(request):
```

Comment:
While passing paths through the `tz_path` fixture via indirect parameterization is clever, it's quite obscure and it's not at all clear what this does without reading the documentation and/or knowing some pretty arcane `pytest` lore.

How about just refactoring this into a function that you call, or refactoring the tests into a class (not a `TestCase`) with a method like `self.path`?

Reply:
Refactored into a plain function.